### PR TITLE
Rename binaries and docker images

### PR DESCRIPTION
**Pull request by Kailai-Wang**
### Context

This PR:
- renames the binaries to `litentry-*` and docker images
- adjust scripts 
- include `litentry dev` in the authors too
- didn't touch the unused part (e.g. teeracle)

### CODE REVIEW

**Review by BillyWooo — APPROVED** [review]
Thanks. LGTM.

**Comment by BillyWooo** [comment]
Besides, got a question: how about the images pushed to hub.docker.com?

File: pallets/sidechain/Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 [package]
 name = "pallet-sidechain"
-description = "Pallet for integritee sidechains"
+description = "Pallet for litentry sidechains"
 version = "0.9.0"
-authors = ["Integritee AG <hello@integritee.network>"]
-homepage = "https://integritee.network/"
-repository = "https://github.com/integritee-network/pallets/"
+authors = ['Litentry Technologies GmbH <info@litentry.com> and Integritee AG <hello@integritee.network>']
```
Comment (kziemianek) [inline pallets/sidechain/Cargo.toml]: This is already an array so maybe `['Litentry Technologies GmbH <info@litentry.com>', 'Integritee AG <hello@integritee.network>']` ?

**Review by kziemianek — APPROVED** [review]
One small comment, the rest looks fine.

File: pallets/sidechain/Cargo.toml (hunk 1 above)
Comment (Kailai-Wang) [reply pallets/sidechain/Cargo.toml]: Thanks, this is indeed cleaner

**Pull request merged**